active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+Gets or sets the ability to scroll horizontally/vertically through user input. Defaults to Enabled.
+
+When set to Disabled, the user will not be able to scroll through user input.
+
+When HorizontalScrollMode is set to Enabled, the user will be able to pan horizontally with touch and a pen. 
+
+The user can also scroll horizontally with the mouse wheel and scroll bar as long as the ScrollableWidth property is strictly positive.
+
+When VerticalScrollMode is set to Enabled, the user will be able to pan vertically with touch and a pen. 
+
+The user can also scroll vertically with the mouse wheel and scroll bar as long as the ScrollableHeight property is strictly positive.
+
+### ScrollingView.IgnoredInputKind property
+Gets or sets the kinds of user input the control does not respond to. Defaults to Non
+
+Each input kind affects both scrolling and zooming capabilities. For instance, if the Pen flag is included, the user can no longer scroll nor zoom using the pen.
+
+
+### ScrollingView.HorizontalScrollChainMode (and VerticalScrollChainMode) property
+
+Gets or sets the ability to chain horizontal scrolling to an outer scroller. Defaults to Auto.
+
+The outer scroller that picks up the scrolling through chaining can be either DirectManipulation-driven or InteractionTracker-driven.
+
+When set to Auto, the `ScrollingView` chains to the closest scrollable outer scroller if any.
+
+When set to Always, the `ScrollingView` pretends there is a scrollable outer scroller and does no overpan.
+
+When set to Never, the `ScrollingView` pretends there is no scrollable outer scroller and overpans.
+
+
+### ScrollingView.ZoomChainMode property
+
+Gets or sets the ability to chain zooming to an outer scroller. Defaults to Auto.
+
+The outer scroller that picks up the zooming through chaining can be either DirectManipulation-driven or InteractionTracker-driven.
+
+When set to Auto, the `ScrollingView` chains to the closest zoomable outer scroller if any.
+
+When set to Always, the `ScrollingView` pretends there is a zoomable outer scroller and does no overzoom.
+
+When set to Never, the `ScrollingView` pretends there is no zoomable outer scroller and overzooms.
+
+### ScrollingView.HorizontalAnchorRatio (and VerticalAnchorRatio) property
+
+Gets or sets ratio within the viewport where the anchor element is selected. Defaults to zero.
+
+When the HorizontalAnchorRatio is NaN, no horizontal anchoring is performed.
+
+When it is 0.0, the left-most element in the viewport is being anchored horizontally.
+
+When it is 1.0, the right-most element in the viewport is being anchored horizontally.
+
+Any number in between is valid. For instance, if it's 0.5, the element in the middle of the viewport is being anchored.
+
+
+### ScrollingView.ScrollFrom method
+
+Asynchronously adds scrolling inertia.
+
+When an inertiaDecayRate is provided to ScrollFrom, it is applied during the duration of the scroll and then reverts back to the default (0.95, 0.95).
```

Comment:
1.	No
2.	No
3.	Yes passing null is the same as (0.95, 0.95)
4.	There is no ScrollingView.ScrollInertiaDecayRate property yet, but will be in the future for inertia in general (like touch driven inertia). But that property will not change temporarily. 

Your expectation is matches our implementation, except that ScrollingView.ScrollInertiaDecayRate is not public yet.

Reply:
Follow-up for (3): Does it mean that if I call `ScrollFrom(inertialDecayRate: 0.5)`, and before that scroll completes, I do another `ScrollFrom(inertialDecayRate: null)`, then the second call will reuse the `0.5` decay rate from the already-active `ScrollFrom`?

Reply:
No, the second operation will use the default 0.95 value because of the 'null' use.   Thanks for asking because looking at our current implementation I realized that we have a bug there. I will open a new GitHub issue for it.

Reply:
Discussion: Suggested text isn't bad, but also the custom inertial decay rate stays until the ScrollFrom completes, or a new ScrollFrom is called. If the scroll is interrupted by a user action, it maintains the custom decay.

Recommend: Is this good behavior? Update the description and/or evaluate the behavior and implementation.